src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/** Lists all persons in the address book to the user.
+ *
+ */

```

Comment:
Look at our [coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments) on how to format doc comments.